provider/resource_rediscloud_subscription_peering_test.go

```diff
@@ -13,6 +13,8 @@ import (
 
 func TestAccResourceRedisCloudSubscriptionPeering_aws(t *testing.T) {
 
+	testAccRequiresEnvVar(t, "EXECUTE_TESTS")
```

Comment:
having now gone through a few of these, it feels like we could extract out these magic strings to some constants just incase you mistype something.

Reply:
Yes, agreed and I will address this in the next PR.